lib/cli/commands/subscribealerts.ts

```diff
@@ -0,0 +1,72 @@
+import { ServiceError, status } from 'grpc';
+import { Arguments, Argv } from 'yargs';
+import { XudClient } from '../../proto/xudrpc_grpc_pb';
+import * as xudrpc from '../../proto/xudrpc_pb';
+import { setTimeoutPromise } from '../../utils/utils';
+import { loadXudClient } from '../command';
+import { AlertType } from '../../constants/enums';
+
+export const command = 'subscribealerts [--pretty]';
```

Comment:
I'm not so sure about breaking convention here and having "non-pretty" be the default. We have the `--json` or `-j` flag on all xucli commands to use the non-pretty json output, I think we should stick to that here. If the `--json` flag is true, we print json, otherwise we print the formatted output.